### DISCUSSION TITLE
Why presets on Directus collections are limited to Activity, Files and Users?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#15779] [Archive]
I'd like to preset Directus_roles

**Comment by br41nslug** [comment]
This is because only Activity, Files and Users have pages with a layout you can make a Preset for. If you were to set a preset for directus_roles there wouldn't be a page where you could see this preset in action.

**Reply by adussarps** [reply]
Would you know how to see the preset on those pages? (users/files..) I have set some, but they seem impossible to display.
